src/replicon_core/dont_replicate.rs

```diff
@@ -80,12 +80,16 @@ mod tests {
 
     #[test]
     #[should_panic]
-    fn without_replication() {
```

Comment:
I removed old test because it can be called without replication now.
Diff shows new test that replaced the old one.